### PR TITLE
Create card block

### PR DESCRIPTION
**Pull request by AnyaLyaLya**
- [DEMO LINK](https://anyalyalya.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://anyalyalya.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,62 @@
...
+        <p class="card__price-label">Price:</p>
+        <p class="card__price">$2,199</p>
+      </div>
+
+      <div class="card__button">
+        <a href="#"
+          class="card__buy"
+          data-qa="hover"
+          required
+        >
```
Comment (Evgenb000) [inline src/index.html:56]: you must move the attribute to a new line when there are more than two
```
<a
  href="#"
  class="card__buy"
  data-qa="hover"
  required
>

Comment (Evgenb000) [inline src/index.html:56]: you must have attributes on the new form when there are more than two

**Review by Evgenb000 — COMMENTED** [review]
Good job!

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,62 @@
...
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style/index.css">
+
```
Comment (VladVeklenko) [inline src/index.html:15]: You can use google fonts linls only once in your code.  Something like that   
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,62 @@
...
+        <p class="rating__review">Reviews: 5</p>
+      </div>
+
+
+      <div class="card__price-box">
+        <p class="card__price-label">Price:</p>
+        <p class="card__price">$2,199</p>
+      </div>
+
+      <div class="card__button">
```
Comment (VladVeklenko) [inline src/index.html:51]: You can make your code more simple without `<di>` , you don't need it here

File: src/style/rating.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+.stars--1 > .stars__star:nth-child(-n + 1),
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 5) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.rating__review {
```
Comment (VladVeklenko) [inline src/style/rating.css:34]: Maybe, style for review suits more in card.css

File: src/style/rating.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star + .stars__star {
+  margin-left: 4px;
+}
```
Comment (VladVeklenko) [inline src/style/rating.css:24]: Margin-right: 4px will be more simple

**Review by VladVeklenko — COMMENTED** [review]
Overall, good job on completing the task!

File: src/style/cardBlock.css [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+  margin-bottom: 16px;
+}
+
+.card__price-box {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+}
```
Comment (Olekk17) [inline src/style/cardBlock.css:47]: Why do you need these empty lines here?

File: src/style/rating.css [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+.stars--1 > .stars__star:nth-child(-n + 1),
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(-n + 5) {
```
Comment (Olekk17) [inline src/style/rating.css:30]: You can use space (' ') combinator here:
```.stars--1 .stars__star:nth-child(-n + 1),\n
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {```

**Review by AlexRog01 — COMMENTED** [review]
Well done!!!

**Pull request closed without merging**